test/integration/asset_manager_test.rb

```diff
@@ -104,6 +104,52 @@ class CreatingAPersonImage < ActiveSupport::TestCase
     end
   end
 
+  class RemovingAPersonImage < ActiveSupport::TestCase
+    setup do
+      @filename = 'minister-of-funk.960x640.jpg'
+      @person = FactoryBot.create(
+        :person,
+        image: File.open(fixture_path.join(@filename))
+      )
+
+      VirusScanHelpers.simulate_virus_scan(@person.image, include_versions: true)
+      @person.reload
+
+      @asset_id = 'asset-id'
+      Services.asset_manager.stubs(:whitehall_asset).returns('id' => "http://asset-manager/assets/#{@asset_id}")
+    end
+
+    test 'removes the person image and all its versions from asset manager' do
+      # Creating a person creates one entry in asset manager for the
+      # uploaded asset and one entry for each of the versions defined
+      # in ImageUploader
```

Comment:
Minor: I found the use of the word "entry" a bit confusing in this comment block. I think it might be clearer to say something like "asset record"...

Reply:
I'll fix that when I rebase, thanks!